zuul.d/projects.yaml

```diff
@@ -120,18 +120,18 @@
         - cluster-api-provider-openstack-image-build:
             branches: master
 
-#- project:
-#    name: terraform-providers/terraform-provider-openstack
-#    periodic:
-#      jobs:
-#        - terraform-provider-openstack-acceptance-test-telefonica:
-#            branches: master
-#        - terraform-provider-openstack-acceptance-test-opentelekomcloud:
-#            branches: master
-#        - terraform-provider-openstack-acceptance-test-orange:
-#            branches: master
-#        - terraform-provider-openstack-acceptance-test-huaweicloud:
-#            branches: master
+- project:
```

Comment:
There is an open issue about terraform jobs cleanup https://github.com/theopenlab/openlab/issues/136 , we can address that issue in this PR too.